SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Utils/UserIdEventArgs.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
```

Comment:
Copyright missing

Reply:
What is the copyright you are referring to? How to add it?

Reply:
Copy from any other cs file.